### DISCUSSION TITLE
embedding_lookup_sparse(wide_w, ids, weights, combiner="sum") C++ api

### DISCUSSION THREAD

**Discussion by songziqin** [apache/incubator-mxnet#14892] [Q&A]
I have read the tensorflow phython code embedding_lookup_sparse(wide_w, ids, weights, combiner="sum") . and i wonder if there is the code for C++ api nn layer and implement the same function. thanks

**Comment by vrakesh** [comment]
@songziqin Thanks for the question, @leleamol  requesting your input on this question.

@mxnet-label-bot  add [Question]
